core/src/com/unciv/logic/trade/TradeEvaluation.kt

```diff
@@ -235,6 +251,13 @@ class TradeEvaluation {
             }
         }
     }
+    fun distanceDiscount(civInfo: CivilizationInfo, capitalcity: CityInfo, city: CityInfo): Int{
+
+        if (abs(capitalcity.location.x - city.location.x) > 500 && abs(capitalcity.location.x - city.location.x) > 500){ // calculates distance between cities
```

Comment:
Don't use this calculation, it doesn't take into account world wrap. Use aerialDistanceTo instead.

Reply:
I already fixed that in a later commit as a change that xlenstra suggested

Reply:
nevermind fixed